src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/custom.py

```diff
@@ -265,6 +265,50 @@ def db_delete_replica_link(  # pylint: disable=too-many-arguments
         link_id=link.name)
 
 
+def db_export(
+        client,
+        database_name,
+        server_name,
+        resource_group_name,
+        **kwargs):
+    keyType = kwargs['storage_key_type']
```

Comment:
Is storage_key_type required? If yes, I think that it can be a regular function param (just like database_name, server_name, etc). If no, then this can throw KeyError and you have to use `keyType = kwargs.get('storage_key_type')` instead